lib/jekyll/commands/serve.rb

```diff
@@ -173,31 +281,51 @@ def enable_logging(opts)
         # forget to add one of the certificates.
 
         private
-        # rubocop:disable Metrics/AbcSize
         def enable_ssl(opts)
-          return if !opts[:JekyllOptions]["ssl_cert"] && !opts[:JekyllOptions]["ssl_key"]
-          if !opts[:JekyllOptions]["ssl_cert"] || !opts[:JekyllOptions]["ssl_key"]
+          jekyll_opts = opts[:JekyllOptions]
```

Comment:
> Why is this necessary?

It's not strictly necessary but it keeps the line length down, makes the code more readable (IMHO), and saves dereferencing the opts variable over and over for the same thing.  I can put it back if you want.